project/api/models.py

```diff
@@ -0,0 +1,79 @@
+"""
+Chron.
+Copyright (C) 2018 Alisa Belyaeva, Ata Ali Kilicli, Amaury Martiny,
+Daniil Mordasov, Liam O’Flynn, Mikhail Orlov.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+from django.db import models
+from polymorphic.models import PolymorphicModel
+
+from .ltree import LtreeField
+
+# Create your models here.
+class PoliticalEntity(models.Model):
+    """
+    A 1-1 mapping between a https://www.wikidata.org/wiki/Q56061, and a PK in our db.
+    Holds an additional color information.
+    """
+
+    wikidata_id = models.IntegerField(unique=True)  # Excluding the Q
+    color = models.IntegerField()
+    admin_level = models.IntegerField()
```

Comment:
If possible, min=0 and max=11

Reply:
>min=0 and max=11 
It's 12 levels in total. I am not sure we should add constrains _before_ we decide and create a documentation about admin levels